File: daySix/Java/MoveZerosToEnd.java
import java.util.Arrays;

public class MoveZerosToEnd{
    
    public static int[] movingZerosToTheEnd(int[] array){
    
        int[] newArray = new int[array.length];
        
        int index = 0;
        
        for(int count = 0; count < array.length; count++){
                
            if (array[count] != 0){
            
                newArray[index] = array[count];
                
                index++;
            
            }
        }
    
        while (index < newArray.length){
        
            newArray[index] = 0;
            
            index++;
        
        }
        
       return newArray; 
    }
        
        
    public static void main(String[] args){
    
    int[] array = {5, 0, 3, 0, 2, 0};
    
    System.out.println(Arrays.toString(movingZerosToTheEnd(array)));
    
    }
    
}
        

File: daySix/Python/move_zeros_to_end.py
def moving_zeros_to_the_end(array):

    new_array = [0] * len(array)

    index = 0

    for count in range(len(array)):
    
        if array[count] != 0:
        
            new_array[index] = array[count]
            
            index += 1

    while index < len(new_array):
    
        new_array[index] = 0
        
        index += 1

    return new_array



